### PR TITLE
134 checkstyle smells

### PR DESCRIPTION
**Pull request by rivalez**

### CODE REVIEW

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java [hunk 1]
```diff
@@ -12,21 +12,28 @@
...
+     * and does not contain invalid characters.
      * Works for IPv4.
      *
      * @param ip
      * @return true if IP is in correct format
      */
+
     @Override
-    public boolean validate(String ip) {
+    public boolean validate(final String ip) {
```
Comment (Ozymadias) [inline battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java]: Do we really need this final everywhere?

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java [hunk 2]
```diff
@@ -12,21 +12,28 @@
 
     private final String digitsOnly = "[0-9]\\d*";
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java]: static

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java [hunk 3]
```diff
@@ -15,16 +15,17 @@
...
- *
  * @since 1.0.1
  */
-public class SocketServer implements Server {
+public final class SocketServer implements Server {
     private static final Logger LOGGER = Logger.getLogger(SocketServer.class.getName());
+    private static final int DEFAULT_WIDTH = 200;
+    private static final int DEFAULT_HEIGHT = 100;
     private ShipClient server;
     private String allMoves = "";
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketServer.java]: just private String allMoves;

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 4]
```diff
@@ -17,29 +17,38 @@
...
 import java.util.Queue;
+//todo: write java doc
 
-
-public class GameScene extends Application {
+/**
+ * JAVA DOC HERE
+ */
+class GameScene extends Application {
+    //todo: naming!!!!
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: todo to remove

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 5]
```diff
@@ -17,29 +17,38 @@
 import javafx.stage.Stage;
 
 import java.util.Queue;
+//todo: write java doc
 
-
-public class GameScene extends Application {
+/**
+ * JAVA DOC HERE
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: now is time for this

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java [hunk 6]
```diff
@@ -17,29 +17,38 @@
 import javafx.stage.Stage;
 
 import java.util.Queue;
+//todo: write java doc
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/GameScene.java]: to remove

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartScene.java [hunk 7]
```diff
@@ -10,17 +10,21 @@
...
     @Override
-    public void start(Stage primaryStage) throws Exception {
+    public void start(final Stage primaryStage) throws Exception {
         Parent start = FXMLLoader.load(getClass().getResource("/startScreen.fxml"));
         Scene scene = new Scene(start);
         primaryStage.setResizable(true);
         primaryStage.setScene(scene);
         primaryStage.show();
     }
+    //todo:write java doc
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartScene.java]: remove

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartScene.java [hunk 8]
```diff
@@ -10,17 +10,21 @@
...
 
-    public static void main(String[] args) {
+    /**
+     * @param args input
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartScene.java]: ???

File: battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java [hunk 9]
```diff
@@ -16,14 +16,21 @@
 
 import java.io.IOException;
 
-public class StartSceneController {
+//todo: write javadoc
```
Comment (rivalez) [inline battleship-client/src/main/java/com/academy/solid/nie/client/ui/StartSceneController.java]: again

File: battleship-server/src/main/java/com/academy/solid/nie/config/language/CommunicateProvider.java [hunk 10]
```diff
@@ -4,5 +4,11 @@
  * It provides the proper communicates based on specified language.
  */
 public interface CommunicateProvider {
+    //todo:
```
Comment (rivalez) [inline battleship-server/src/main/java/com/academy/solid/nie/config/language/CommunicateProvider.java]: god pls no

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java (hunk 2 above)
Comment (Ozymadias) [reply battleship-client/src/main/java/com/academy/solid/nie/client/communication/IpValidator.java]: I told you about that - it shouldn't be static. Only tools wants it to be

File: battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketClient.java [hunk 11]
```diff
@@ -35,16 +35,18 @@ public boolean run() throws IOException {
 
         String fromServer;
         while (!("Provide ships").equals(fromServer = in.readLine())) {
-            if (LOGGER.isLoggable(Level.INFO))
+            if (LOGGER.isLoggable(Level.INFO)) {
                 LOGGER.info("Server: " + fromServer);
-            if (fromServer.equals("Game has started. 1"))
+            }
+            if (fromServer.equals("Game has started. 1")) {
```
Comment (smolamarcin) [inline battleship-client/src/main/java/com/academy/solid/nie/client/communication/SocketClient.java]: Maybe you should extract it as a static field?

**Pull request merged**